muikku-core-plugins/src/main/resources/META-INF/resources/scripts/gui/workspace.js

```diff
@@ -73,25 +65,14 @@
   };
 
   $(document).ready(function() {
-    $('#staticNavigationWrapperWorkspace').waypoint('sticky', {
-      stuckClass : 'stuckStNav'
-    });
-  
-    // Workspace navigation
-    if ($('#workspaceNavigationWrapper').length > 0) {
-      refreshNavigationWrapperPosition();
-      $(window).resize(function(){
-        refreshNavigationWrapperPosition();
-      });
-    }
-    
-    if ($('.wi-workspace-dock-navi-button-cancel-evaluation'.length > 0)) {
-      $('.wi-workspace-dock-navi-button-cancel-evaluation').hide();
+
+    if ($('.workspace-dock-navi-button-cancel-evaluation'.length > 0)) {
```

Comment:
1) This evaluates always as true 2) What is the purpose of this if? ($('.workspace-dock-navi-button-cancel-evaluation') is a set, so if there are no elements, hide just does not do anything)